MachObfuscator/Commons/Paths.swift

```diff
@@ -1,6 +1,23 @@
+import Foundation
+
 enum Paths {
     static let separator: Character = "/"
-    static let iosRuntimeRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/Library/CoreSimulator/Profiles/Runtimes/iOS.simruntime/Contents/Resources/RuntimeRoot"
     static let macosFrameworksRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk"
     static let iosFrameworksRoot = "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS.sdk"
+
+    static let iosRuntimeRoot: String = {
+        guard let iosRoot = possibleIosRuntimeRoots.first(where: FileManager.default.fileExists(atPath:)) else {
```

Comment:
you choose runtime according to version which is installed, not version which was used to compile the application. I am not sure if this is a problem.
BTW, if user has both Xcode 10 and 11 this always chooses one that is installed as default `Xcode.app`.

Reply:
All you say is true. This is kind of minimum to run MachObfuscator using latest Xcode.

Regarding choosing right Xcode version – you can never be sure about the version of Xcode chosen to compile the binary. We could try to detect `LC_LOAD_DYLIB` library versions, but we would have problems with:
- static libraries – linked within other libraries by the same Xcode that was used to compile the library, but possibly compiled with other Xcode version,
- dynamic libraries – possibly compiled by other Xcode version than the main app.
That would possibly require a lot of different Xcode versions.

Regarding symbols, only the Obj-C is an issue here. One latest runtime will never be sufficient, because:
- it can contain no information about app methods that are obsolete, but are still used by older iOS versions,
- some symbols are being concatenated from strings, it is not possible to recover them from runtime.

So, if just the Obj-C is an issue, we should parse the framework headers... and we already do this. So we should always try to use the latest available Xcode version to parse the public headers.

I assumed `/Applications/Xcode.app` is the latest Xcode version. Yes – I install Xcode from AppStore, and yes – I keep beta Xcode in separate location. I assumed the `/Applications/Xcode.app` always contain the latest stable Xcode. Now I think I was wrong.

What do you think about using `xcode-select` to get currently selected Xcode?

Reply:
I do not have much experience with Xcode versions, but I think there are valid cases when one may want to have multiple versions installed. It may be for local developement, but probably it might be more important for continuous integration, where there is higher probability of having multiple Xcode versions installed. fastlane has nice support for this case (via `xcversion`).

For me,  using the same Xcode version for building the app and for gathering symbols to blacklist is a safest approach, but using newest or default Xcode for the second part probably will also work (unless someone has setup with old Xcode being the default one).

I think that it will be most common case to build and obfuscate the application on the same machine. Detecting Xcode version from binaries is hard or impossible. So I see 2 possibilities here:

1) MachObfuscator using xcode-select to determine path. I am not sure if this will work with `xcversion` in fastlane (docs are very sparse). Fastlane has also another command `xcode_select` that by definition should change default Xcode path.
2) pass path to Xcode as a parameter to Obfuscator with default being `/Applications/Xcode.app` or value obtained from `xcode-select`. This is more explicit than the first option.

Reply:
Sorry for leaving this PR for a long time. I decided to create an issue to choose the Xcode version (#93). I have also checked the `xcversion` [sources](https://github.com/fastlane/fastlane/blob/e0db2e6c728784328329b8182ff0acada6395b6e/fastlane/lib/fastlane/actions/xcversion.rb#L14), looks like it uses the `DEVELOPER_DIR` mentioned by the `man xcode-select`. Will try to touch this as soon as possible.